docs/guides/cli/kubectl-plugin/index.md

```diff
@@ -1,432 +1,331 @@
-
-> Note: You must provide `<volumenName>:<mountPath>:<subPath>` in the `--volume-mounts` flag to specify the volumes and their mountPath and subPath. The `:<subPath>` part is optional.
+### Copy VolumeSnapshot
 
-**Format:**
+To copy a VolumeSnapshot, you need to provide `VolumeSnapshot` name and destination namespace. You will provide the destination namespace by using flag. The available flags are:
 
-```bash
-kubectl stash create restoresession <restoresession-name> [flags]
-```
+| Flag             | Description                                                                    |
+|------------------|--------------------------------------------------------------------------------|
+| `--namespace`    | Indicates the namespace of the respective `VolumeSnapshot`.                    |
+| `--to-namespace` | Indicates the destination namespace where the `VolumeSnapshot` will be copied. |
 
 **Example:**
 
 ```bash
-$ kubectl stash create restoresession ss-restore --namespace=demo --repo-name=gcs-repo --target-apiversion=apps/v1 --target-kind=StatefulSet --target-name=stash-recovered --paths=/source/data --volume-mounts=source-data:/source/data
+$ kubectl kubestash cp volumesnapshot my-vol-snap --namespace=demo --to-namespace=demo1
 ```
 
-## Copy Command
-
-`kubectl stash cp` command is used to copy stash objects from one namespace to another namespace. It copies various objects like `Secret`, `Repository`, `BackupConfiguration` and `VolumeSnapshot` etc.
+## Clone PVC
 
-### Copy Secret
+`kubectl kubestash clone pvc` command is used to clone PVC from one namespace to another namespace. When we run the command the cloning process consists of the following steps:
 
-To copy a Secret, you need to provide Secret name and destination namespace. You will provide the destination namespace by using flag. The available flags are:
+- At first, It creates a BackupStorage in the source namespace (if no storage is provided).
+- It creates a RetentionPolicy in the source namespace.
+- Using these BackupStorage and RetentionPolicy, it creates a BackupConfiguration targeting the PVC to take backup.
+- After the backup process succeeded, It creates a RestoreSession in the source namespace with the target in destination namespace.
+- finally, It restores the backed up data into VolumeClaimTemplate in the destination namespace.
 
-| Flag             | Description                                                          |
-| ---------------- |----------------------------------------------------------------------|
-| `--namespace`    | Indicates the namespace of the respective `Secret`.                  |
-| `--to-namespace` | Indicates the destination namespace where the `Secret` will be copied. |
+To clone a PVC, you need to provide backend credentials for creating BackupStorage (if no storage is provided) and encryption secret.
+You will provide the backend credential and encryption secret by using flags. The available flags are:
+
```

Comment:
To clone a PVC, you can provide the reference of the existing BackupStorage if you already have one; otherwise, you have to provide the backend information. Additionally, you need to provide the encryption secret reference. The available flags are:

Reply:
I think we can skip taking the encryption secret from the user.